kmmo/README.md

```diff
@@ -24,6 +24,21 @@ Follow the installation guide below to install the KMM operator via CLI or web c
 # Canary deployment with KMM
 Canary deployment is enabled by default to deploy the driver container image only on specific node(s) to ensure the initial deployment succeeds prior to rollout to all the eligible nodes in the cluster. This safety mechanism can reduce risk and prevent a deployment from adversely affecting the entire cluster.
 
+# Optional: Set alternative firmware path at runtime with KMM
+Instead of using a [machine config to set alternative firmware path](/machine_configuration/README.md#set-up-an-alternative-firmware-path-for-the-cluster) which triggers reboot, follow the steps below to set the firmware path at runtime.
```

Comment:
I'd like to take it as the only option for users to setup the firmware path, and remove the setting from MCO.

Reply:
Sounds good. I can update this PR to remove optional. We can have a separate PR to remove the setting from MCO.

Reply:
@hershpa  is there any elegant way to only set the `setFirmwareClassPath: /var/lib/firmware` instead of redeploying the whole configmap? this looks like a workaround.. Totally understand that we are trying to modify the value of the configmap which is the yaml file in this case.. @mregmi do you have any suggestions here?

Reply:
Unfortunately, we have not found a simpler way yet. We are definitely open to updating it. Typically, if we want to be more elegant, we would need to use something like yq but that would be another binary the user would need to install.

Reply:
> Sounds good. I can update this PR to remove optional. We can have a separate PR to remove the setting from MCO.

Thanks! :-)

Reply:
PR updated!